### ISSUE TITLE
Ability to ensure support for a unicode version

### ISSUE THREAD

**Issue by AlexDaniel** [rakudo/rakudo#2031]
Wouldn't it be right to have something like this?
```
use Unicode :ver(v42..*);
```

The reason is that unicode support is rolling without any connection to, for example, language versions. So under `use v6.c` or `use v6.d` you can't know which unicode version is supported and your code can behave differently on different compiler versions/compilers (if you rely on new unicode features). I *think* that all perl 6 language versions also guarantees a minimum unicode version that must be supported, but that doesn't help that much unless we release perl 6 faster than new unicode versions are released.

I understand that supporting more than one unicode version at the same time (depending on which `use Unicode …` is used) is probably too difficult and is likely an overkill anyway, but maybe we need to provide a way to ensure that at least version X is supported by the compiler?

ping @samcv

**Comment by samcv** [comment]
I agree there should be some way to guarentee the Unicode version. I'm not sure what the syntax should be though. What do other people think?

**Comment by lizmat** [comment]
I think a start would be to be able to find out which unicode version is currently active / supported.  Can we do that already?

**Comment by samcv** [comment]
@lizmat no we don't have that feature yet. Would we want to add that onto some pre-existing op or does it need a new one?

**Comment by lizmat** [comment]
Which pre-existing op where you thinking about?  Feels like a new op to me?  Or do you intend to keep different versions for different codepoints?

**Comment by samcv** [comment]
No, I don't intend to keep different versions, but there may be a VM info op? Though maybe we want a separate unicode version op? @jnthn what are you thoughts on this?

**Comment by lizmat** [comment]
ah, maybe we can lift it from the compiler info already?
